mlrun/utils/helpers.py

```diff
@@ -563,14 +563,38 @@ def get_parsed_docker_registry() -> Tuple[Optional[str], Optional[str]]:
         )
 
 
-def pr_comment(repo: str, issue: int, message: str, token=None):
```

Comment:
If you're already touching all this code, maybe move it to `mlrun/utils/notifiers.py`

Reply:
will refactor in next PR, i want to turn the slack, pr, .. to classes so will be easy to add email, sms, ..